.env.example

```diff
@@ -0,0 +1,45 @@
+# Spyglass Docker Compose Configuration
```

Comment:
The current established norm is to describe how to use a config file in a doc, and then leave the config file itself sparse. This gives the benefit of providing a renderable markdown for folks poking around in github

I suggest moving instructions to a another file like README / QUICKSTART / 00_Setup